tasks/build/archives.js

```diff
@@ -13,13 +13,13 @@ module.exports = function createPackages(grunt) {
 
   let archives = async (platform) => {
     // kibana.tar.gz
-    await exec('tar', ['-zchf', platform.tarPath, platform.buildName]);
+    await exec('tar', ['-zchf', relative(buildPath, platform.tarPath), platform.buildName]);
 
     // kibana.zip
     if (/windows/.test(platform.name)) {
-      await exec('zip', ['-rq', '-ll', platform.zipPath, platform.buildName]);
```

Comment:
You should build kibana using **Cygwin**. You must install all the packages used for the build (the default ones plus **zip**, **tar** and **shasum**).